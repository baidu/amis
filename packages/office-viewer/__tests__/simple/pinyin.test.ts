import {snapShotTest} from '../snapShotTest';

test('bold', async () => {
  snapShotTest('./docx/simple/pinyin.docx');
});
