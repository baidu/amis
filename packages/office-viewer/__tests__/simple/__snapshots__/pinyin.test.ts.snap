// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pinyin 1`] = `
<article>
  <section
    style="position: relative; width: 815.98px; padding: 96.00px 96.00px 96.00px 96.00px;"
  >
    <p
      class="docx-viewer-0-p"
      style="position: relative;"
    >
      <span
        class="docx-viewer-0-r"
      >
        <ruby>
          <span
            class="docx-viewer-0-r"
          >
            拼
          </span>
          <rp />
          <rt>
            <span
              class="docx-viewer-0-r"
              style="font-family: DengXian; font-size: 6.67px;"
            >
              pīn
            </span>
          </rt>
          <rp />
        </ruby>
      </span>
      <span
        class="docx-viewer-0-r"
      >
        <ruby>
          <span
            class="docx-viewer-0-r"
          >
            音
          </span>
          <rp />
          <rt>
            <span
              class="docx-viewer-0-r"
              style="font-family: DengXian; font-size: 6.67px;"
            >
              yīn
            </span>
          </rt>
          <rp />
        </ruby>
      </span>
    </p>
  </section>
</article>
`;
